CosmosContracts/juno-tools
# feat(pages): update home page

Fixes #11 #72

## Description

Based on this feedback[^1], this PR adjusts the home page and adds the brand font family.

## Changes

List any technical changes.

- [x] add brand/heading font family assets
- [x] update home page
- [x] update heading font families
- [x] extract and use home card component

## Screenshots

<img width="1492" alt="image" src="https://user-images.githubusercontent.com/8220954/157100812-073b5791-c7e3-4dbd-b1b0-509073d49d17.png">


## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- open home page `/` and see if the links below is accessible and clickable

## Links

\-

[^1]: https://discord.com/channels/927644555166236712/933515307597848656/949425923223072788
